RealmSwift/Combine.swift

```diff
@@ -35,15 +35,18 @@ import Realm.Private
 ///
 /// You can also manually conform to `Identifiable` if you wish, but note that
 /// using the object's memory address does *not* work for managed objects.
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
```

Comment:
should these guards not match `RealmSubscribable `?

```suggestion
@available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, *)
```

Reply:
I simply forwarded the @available attribute from `Identifiable` itself:
<img width="405" alt="Screenshot 2021-02-05 at 09 43 30" src="https://user-images.githubusercontent.com/4356928/107017085-9e171400-6796-11eb-9093-5327e188ff00.png">
I'd say `macOS` is more correct than `OSX` at this point as well?

As far as the `_ApplicationExtension`s, Combine and SwiftUI don't seem to explicitly mark them.

Reply:
True, I guess forwarding the message from Apples API is the correct thing here anyway. Marking as resolved.